pkg/controller/nodeipam/ipam/cloud_cidr_allocator_test.go

```diff
@@ -970,240 +1004,39 @@ func TestUpdateCIDRAllocation(t *testing.T) {
 			},
 			nodeChanges: func(node *v1.Node) {},
 		},
-	}
```

Comment:
the diff goes absolutely nuts here. Its probably better to view this file in an editor.

This is where I was able to delete the duplicated test from #534